README.md

```diff
@@ -157,6 +161,33 @@ To override the **sourceDirectory**, **outputDirectory**, recurse over **sourceD
 </plugins>
 ```
 
+To override the **namespaceMapping** of Avro protocols under the `com.example.packagename` namespace to `com.example.packagenamechanged`
+
+```xml
+<plugins>
+    <plugin>
+        <groupId>at.makubi.maven.plugin</groupId>
+        <artifactId>avrohugger-maven-plugin</artifactId>
+        <executions>
+            <execution>
+                <phase>generate-sources</phase>
+                <goals>
+                    <goal>generate-scala-sources</goal>
+                </goals>
+            </execution>
+        </executions>
+        <configuration>
+            <namespaceMapping>
+                <Mapping>
```

Comment:
For convenience, can you change the example doc to `mapping` with a lower case? I guess this should work with the upper case `Mapping` class name.